backend/src/index.ts

```diff
@@ -384,6 +386,19 @@ handlers[API_ANIMATION_VIDEO] = async (input) => {
   return {response: new Response(result, responseOptions(CONTENT_TYPE_VIDEO_MP4))};
 };
 
+handlers[API_PROFILE_AVATAR] = async (input) => {
+  const result = await db.get(`profile/avatar:${expectUuidParam(input, "id")}`, "arrayBuffer");
+  // TODO how to handle content type?
+  return {response: new Response(result, responseOptions(CONTENT_TYPE_VIDEO_MP4))};
+};
+
+handlers[API_PROFILE_AVATAR_CREATE] = async (input) => {
+  const id = uuid();
+  const base64ImageData = input;
+  const result = await db.put(`profile/avatar:${id}`, base64ImageData);
```

Comment:
Lets do `user/avatar` just so we know everything associated with a user starts with  `user`.

Reply:
Sounds good thanks, I'd like to keep the keys as consistent as possible.